jetson/power_logging/model/benchmark.py

```diff
@@ -113,37 +110,37 @@ def benchmark(args: argparse.Namespace) -> None:
         model = model.to(dtype)
         print(f"Using {DEVICE=} for benchmarking")
         if DEVICE == "cpu":
-            print("WARNING: Running on CPU. Timing may vary")
+            print("Warning: Running on CPU.")
 
-        print("Warm up ...")
         st = time.perf_counter()
+        print("Warm up ...")
         with torch.no_grad():
             for _ in range(args.warmup):
                 _ = model(input_data)
         print(f"Warm complete in {time.perf_counter()-st:.2f} sec ...")
 
-        print("Start timing using pytorch backend ...")
+        print("Starting timing inference ...")
         latencies = []
         start_events = [CudaEvent(enable_timing=True) for _ in range(args.runs)]
         end_events = [CudaEvent(enable_timing=True) for _ in range(args.runs)]
-
+        
         with torch.no_grad():
             for i in tqdm(range(args.runs)):
                 start_events[i].record()
                 _ = model(input_data)
                 end_events[i].record()
 
-                if torch.cuda.is_available():
+                if torch.cuda.is_available:
```

Comment:
Needs to be a function call
```suggestion
                if torch.cuda.is_available():
```

Reply:
fixed